spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -4,6 +4,15 @@
 include Features::LocalizationHelper
 include Features::ActiveJobHelper
 
+describe "redirect to Users::RegistrationController" do
```

Comment:
I forgot the code doesn't actually touch the controller ... so this is the wrong directory for this.

I think this is a good candidate for a routes spec: https://www.relishapp.com/rspec/rspec-rails/docs/routing-specs/route-to-matcher

So in the spirit of adding test files in places that match the location in the app, I'd add a new `spec/routing/routes_spec.rb` (just one file for now, since we don't have that many routes)